src/examples/src/LinkedCard.m.css

```diff
@@ -0,0 +1,37 @@
+.root {
+	position: relative;
+	flex-grow: 1;
+	display: flex;
+}
+
+.card {
+	display: flex;
+	flex-grow: 1;
+}
+
+.link,
+.link:link,
+.link:visited {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	text-decoration: none;
+	display: flex;
+	transition: box-shadow 0.4s ease;
+	width: 100%;
+	z-index: 1;
+}
+
+.link:hover {
+	box-shadow: 0 1px 0px rgba(0, 0, 0, 0.05), 0 2px 15px rgba(0, 0, 0, 0.05), 0 28px 50px rgba(0, 0, 0, 0.16);
```

Comment:
Should these px values be in reference to grid size?

Reply:
I'm not sure that makes too much sense for `box-shadow` especially when the values are so specific? I might be wrong. Either way maybe it should be a CSS variable.

Reply:
FYI I don't think we used any css variables when we set up the examples app.